thirdparty/cJSON/cJSON.c

```diff
@@ -80,10 +80,8 @@ CJSON_PUBLIC(const char *) cJSON_GetErrorPtr(void)
 }
 
 CJSON_PUBLIC(char *) cJSON_GetStringValue(cJSON *item) {
-    if (!cJSON_IsString(item)) {
```

Comment:
Why did you remove the braces?  True, they are not required in C, but they don't hurt anything, and are arguably better coding style.

It's always a pain to have to remember to add them later if you find out you need a second statement in the `if` block.